### PR TITLE
Finished form component

### PR DESCRIPTION
**Pull request by H-Richard**
I finished the form component, it now sends an email from and to: noreply.utscproject@gmail.com, submissions must wait 10 seconds in between. 

Fixed a bug where the form goes out of focus initially.

:)

### CODE REVIEW

File: src/components/ContactUsForm/ContactUsForm.js [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+      <textarea className={styles.Message} {...input} placeholder={label} type={type} />
+      :
+      <input className={styles.Regular} {...input} placeholder={label} type={type} />
+    }
+
+    {touched && error && <span>{error}</span>}
+  </div>
+)
+
+
```
Comment (liuzimin) [inline src/components/ContactUsForm/ContactUsForm.js]: remove whitespace

File: src/components/ContactUsForm/ContactUsForm.js [hunk 2]
```diff
@@ -1,13 +1,40 @@
 import React from 'react';
-import { Field, reduxForm, SubmissionError } from 'redux-form';
+import { Field, reduxForm, SubmissionError, reset } from 'redux-form';
+import { submit, successConfirmation } from '../../store/actions/submit';
+import { connect } from 'react-redux';
 import styles from './ContactUsForm.module.css';
 
 // const  { DOM: { textarea } } = React;
```
Comment (liuzimin) [inline src/components/ContactUsForm/ContactUsForm.js]: remove

File: src/components/ContactUsForm/ContactUsForm.js [hunk 3]
```diff
@@ -1,13 +1,40 @@
...
+      <input className={styles.Regular} {...input} placeholder={label} type={type} />
+    }
+
+    {touched && error && <span>{error}</span>}
+  </div>
+)
+
+
+let ContactUsForm = (props) => {
+  console.log("form status", props);
```
Comment (liuzimin) [inline src/components/ContactUsForm/ContactUsForm.js]: remove consolelog

File: src/components/ContactUsForm/ContactUsForm.js [hunk 4]
```diff
@@ -103,17 +116,29 @@ let ContactUsForm = props => {
         label="Message"
       />
       {props.error && <strong>{props.error}</strong>}
```
Comment (liuzimin) [inline src/components/ContactUsForm/ContactUsForm.js]: do not style with html tags. Use css

File: src/store/reducers/submit.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    message: "",
+    loading: false,
+    success: false,
+    timeout: false
+}
+
+const submit = (state = initialState, action) => {
+    switch(action.type){
+        case actionTypes.SUBMISSION_LOADING:
+                console.log("SUBMISSION LOADING");
```
Comment (laiedgar) [inline src/store/reducers/submit.js]: remove console log

File: src/store/actions/submit.js [hunk 6]
```diff
@@ -0,0 +1,35 @@
+import * as actionTypes from './actionTypes';
+import axios from '../../axios-projects';
+
+export const submit = (submission) => {
+    return (dispatch) => {
+        dispatch({
+            type: actionTypes.SUBMISSION_LOADING
+        })
+        axios.post("https://us-central1-utsc-projects.cloudfunctions.net/sendmail", submission)
```
Comment (laiedgar) [inline src/store/actions/submit.js]: should put urls in global file

File: src/components/ContactUsForm/ContactUsForm.js [hunk 7]
```diff
@@ -1,13 +1,40 @@
 import React from 'react';
-import { Field, reduxForm, SubmissionError } from 'redux-form';
+import { Field, reduxForm, SubmissionError, reset } from 'redux-form';
+import { submit, successConfirmation } from '../../store/actions/submit';
+import { connect } from 'react-redux';
 import styles from './ContactUsForm.module.css';
 
 // const  { DOM: { textarea } } = React;
```
Comment (DiefOu) [inline src/components/ContactUsForm/ContactUsForm.js]: remove commented out code

File: src/components/ContactUsForm/ContactUsForm.js (hunk 1 above)
Comment (H-Richard) [reply src/components/ContactUsForm/ContactUsForm.js]: done

File: src/components/ContactUsForm/ContactUsForm.js (hunk 3 above)
Comment (H-Richard) [reply src/components/ContactUsForm/ContactUsForm.js]: done

File: src/store/actions/submit.js (hunk 6 above)
Comment (H-Richard) [reply src/store/actions/submit.js]: done

File: src/components/ContactUsForm/ContactUsForm.js (hunk 7 above)
Comment (H-Richard) [reply src/components/ContactUsForm/ContactUsForm.js]: done

File: src/components/ContactUsForm/ContactUsForm.js (hunk 4 above)
Comment (H-Richard) [reply src/components/ContactUsForm/ContactUsForm.js]: done

**Pull request merged**